README.md

```diff
@@ -2,45 +2,162 @@
 
-For tutorials, samples, quick starts, and other documentation, go to [Azure for C++ Developers](https://azure.github.io/azure-sdk-for-cpp).
+You can find additional information for specific libraries by navigating to the appropriate folder in the `/sdk` directory. See the **README.md** file located in the library's project folder. For example, [here is the doc for the storage client library](https://github.com/Azure/azure-sdk-for-cpp/tree/master/sdk/storage#azure-storage-client-library-for-c).
 
-## Packages available
-Each service might have a number of libraries available from each of the following categories:
-* [Client - New Releases](#client-new-releases)
-* [Client - Previous Versions](#client-previous-versions)
+For API reference docs, tutorials, samples, quick starts, and other documentation, go to [Azure for C++ Developer Docs](https://azure.github.io/azure-sdk-for-cpp).
+
+### Download & Install the SDK
+
+The easiest way to get the C++ SDK is to leverage [vcpkg](https://github.com/microsoft/vcpkg). You'll need to have [Git](https://git-scm.com/downloads) installed before getting started.
+
+First clone and bootstrap vcpkg itself. You can install it anywhere on your machine, but **make note** of the directory where you clone the vcpkg repo.
+
+```cmd
+> git clone https://github.com/microsoft/vcpkg
+```
+
+On Windows:
+
+```cmd
+> .\vcpkg\bootstrap-vcpkg.bat
+```
+
+On Linux:
+
+```sh
+> ./vcpkg/bootstrap-vcpkg.sh
+```
+
+To install the libraries for your project, run the following, optionally specifying the triplet:
+
+```cmd
+> .\vcpkg\vcpkg install [packages to install]
+```
+
+For example, this will install the `x64-windows` triplet. On Windows, not specifying a triplet will default to x86-windows:
+
+```cmd
+> .\vcpkg\vcpkg install azure-storage-blobs-cpp:x64-windows
+```
+
+See the [list of packages](https://github.com/Azure/azure-sdk-for-cpp#vcpkg) available via vcpkg below. All Azure C++ SDK package names start with `azure-`.
+
+You can also search for the libraries you need with the `search` subcommand:
+
+```cmd
+> .\vcpkg\vcpkg search [search term]
+```
+
+For example:
+
+```cmd
+> .\vcpkg\vcpkg search azure-
+```
+
+Once the library is installed, follow the instructions from the console output to include the library in your `CMake` application. For example, to include `azure-storage-blobs-cpp`, add the following to your `CMakeLists.txt` file:
+
+```CMake
+find_package(azure-storage-blobs-cpp CONFIG REQUIRED)
+target_link_libraries(myProject PRIVATE Azure::azure-storage-blobs)
+```
+
+> NOTE: All the Azure client libraries take a dependency on `azure-core-cpp` which provides functionality commonly needed by all Azure clients. When you install any client library via vcpkg, it will bring in all the necessary dependencies as well. You don't need to install those individually to get started.

```

Comment:
```suggestion
> NOTE: All the Azure client libraries take a dependency on `azure-core-cpp` which provides functionality commonly needed by all Azure clients. When you install any client library via Vcpkg, it will bring in all the necessary dependencies as well. You don't need to install those individually to get started.

```